src/client/automation/playback/click/index.js

```diff
@@ -225,12 +225,20 @@ export default class ClickAutomation {
         return cursor
             .buttonUp()
             .then(() => {
-                var prevElementForEvent = this.eventArgs.element;
-
-                this.eventArgs            = this._calculateEventArguments();
-                this.eventState.skipClick = prevElementForEvent !== this.eventArgs.element;
-
-                eventSimulator.mouseup(this.eventArgs.element, this.eventArgs.options);
+                var topElementEventArgs = this._calculateEventArguments();
```

Comment:
It's better to keep 
```js
var prevElementForEvent = this.eventArgs.element;

this.eventArgs            = this._calculateEventArguments();
```

because `this.eventArgs` is used in other methods of the class, so you should keep this as it is to avoid unexpected problems